subsys/net/lib/sockets/sockets.c

```diff
@@ -1852,6 +1852,13 @@ int zsock_getsockopt_ctx(struct net_context *ctx, int level, int optname,
 			}
 			break;
 		}
+	case IPPROTO_TCP:
+		switch (optname) {
+		case TCP_NODELAY:
+			ret = net_tcp_get_option(ctx,
+				TCP_OPT_NODELAY, optval, optlen);
```

Comment:
Could align the arguments here as well, see how it's done above.

Reply:
Fixed